frontend/src/components/common/Dropdown/HiddenDropdown/HiddenDropdown.tsx

```diff
@@ -0,0 +1,27 @@
+import { HTMLAttributes } from 'react';
+
+interface HiddenDropdownProps extends HTMLAttributes<HTMLSelectElement> {
```

Comment:
아래와 같이 작성하면 `HTMLAttributes` 를 import 하지 않아도 사용할 수 있답니다 🫡
```suggestion
interface HiddenDropdownProps extends React.HTMLAttributes<HTMLSelectElement> {
```

Reply:
처음 알았네요! 수정했습니다 👍